corelib-search/src/main/java/eu/europeana/corelib/search/SearchService.java

```diff
@@ -15,139 +12,14 @@
 import java.util.Map;
 
 /**
- * Search service that retrieves BriefBeans or APIBeans in the case of a query
- * calculateTag or a FullBean in the case of a user selection. Currently the
- * implementation uses SOLR for Brief/APIBeans and MongoDB for FullBean
- * retrieval.
+ * Search service that retrieves BriefBeans or APIBeans from Solr
  *
  * @author Yorgos.Mamakis@ kb.nl
  * @author Willem-Jan Boogerd <www.eledge.net/contact>
+ * @author Patrick Ehlert (major refactoring April 2020)
  */
 public interface SearchService {
```

Comment:
same thing as for the RecordService interface - RecordServiceImpl implementation: can we get rid of that while we're at it? It defies the purpose of using this pattern if an interface is only implemented by one class.